### DISCUSSION TITLE
¿La traducción en español es precisa?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#109] [Q&A]
La oración parece ser coloquial, así que estoy en duda si la traducción en español es precisa.

https://github.com/rdrg109/subtitles-npcr/blob/0f3f9c4fc1edc871087c034f7faaeb8458b4403c/sentences/44-1.yaml#L326-L330

**Comment by rodrigo-morales-1** [comment]
I'm converting this issue into a discussion, because there is no request to modify any of the files in the repository, but instead this is a question.

**Comment by fhxshrs** [comment]
是不是应该加上"una vez" ? 我也不太确定你说的不准确是指什么。

**Reply by rodrigo-morales-1** [reply]
@fhxshrs 英语翻译和西语翻译表达同样的意思，所以如果你觉得英语翻译是准确的，西语也是

你觉得英语翻译是准确的吗？

**Reply by fhxshrs** [reply]
我觉得还算准确，你也可以把你建议的翻译发出来我看一下。
